src/i18n.jsx

```diff
@@ -24,22 +21,32 @@ const setLanguage = langCode => {
   currentLanguage.setValue(langCode);
 };
 
-const loadLocaleData = locale => {
+const loadLocaleData = async locale => {
+  let localeToLoad;
   switch (locale) {
-    case 'nl':
-      return messagesNL;
-    default:
-      return messagesEN;
+    case 'nl': {
+      localeToLoad = 'nl';
+      break;
+    }
+    case 'en':
+    // in case (accidentally) a locale is set that we don't ship translations for yet,
+    // fall back to English.
+    // eslint-disable-next-line no-fallthrough
+    default: {
+      localeToLoad = 'en';
+    }
   }
```

Comment:
this may look a bit over-engineered, but our backend *does* support things like `nl_NL` and `nl_BE` locales and doing things this way makes it future proof for languages we may add in the future that have regional variants.